src/digitRecognition/ErrorFunction.java

```diff
@@ -0,0 +1,232 @@
+     * @return  result - true if ok, false if something goes wrong
+     *******************************************************************************/
+    boolean randomClassifier() {
+        for (ImageIDX aTestCol : testCol) {
+            Random random = new Random();
+            int rand = random.nextInt(10);
+            switch (rand) {
+                case 0: {
+                    imagesByClass.zero.add(aTestCol);
+                    break;
+                }
+                case 1: {
+                    imagesByClass.one.add(aTestCol);
+                    break;
+                }
+                case 2: {
+                    imagesByClass.two.add(aTestCol);
+                    break;
+                }
+                case 3: {
+                    imagesByClass.three.add(aTestCol);
+                    break;
+                }
+                case 4: {
+                    imagesByClass.four.add(aTestCol);
+                    break;
+                }
+                case 5: {
+                    imagesByClass.five.add(aTestCol);
+                    break;
+                }
+                case 6: {
+                    imagesByClass.six.add(aTestCol);
+                    break;
+                }
+                case 7: {
+                    imagesByClass.seven.add(aTestCol);
+                    break;
+                }
+                case 8: {
+                    imagesByClass.eight.add(aTestCol);
+                    break;
+                }
+                case 9: {
+                    imagesByClass.nine.add(aTestCol);
+                    break;
+                }
+
+                default: {
+                    System.out.println("Random value incorrect");
+                    return false;
+                }
+            }
+        }
+        System.out.println("Random classify complete\n");
+        accordanceAnalyse(imagesByClass);
+
+        return true;
+    }
+
+
+
+
+
+    /********************************************************************************
+     *
+     * ImagesByClass is a class for ten collections of images
+     * Used for classify images by their content
+     *
+     *  zero - is a collection of images where draw 0
+     *  one - is a collection of images where draw 1
+     *  two - is a collection of images where draw 2
+     *  three - is a collection of images where draw 3
+     *  four - is a collection of images where draw 4
+     *  five - is a collection of images where draw 5
+     *  six - is a collection of images where draw 6
+     *  seven - is a collection of images where draw 7
+     *  eight - is a collection of images where draw 8
+     *  nine - is a collection of images where draw 9
+     *******************************************************************************/
+    class ImagesByClass {
+        ArrayList<ImageIDX> zero = new ArrayList();
+        ArrayList<ImageIDX> one = new ArrayList();
+        ArrayList<ImageIDX> two = new ArrayList();
```

Comment:
It may be better to have one list of lists than multiple lists. What if you need to recognize more numbers?
List<List<ImageIDX>> and then take a concrete list by index.